cloud/metrics.md

```diff
@@ -23,3 +32,73 @@ In some cases, gray vertical bars display on the metrics dashboard, like this:
 
 This indicates that metrics have not been collected for the period shown. It
 does not mean that your Timescale Cloud service was down.
+
+# Query-level statistics with pg_stat_statements
+The `pg_stat_statements` extension gives you query-level statistics for your SQL
+statements. It comes pre-installed with Timescale Cloud.
+
+<highlight type="note">
+For more information about `pg_stat_statements`, see the 
+[PostgreSQL documentation](https://www.postgresql.org/docs/current/pgstatstatements.html).
+</highlight>
+
+## Query the pg_stat_statements view
+You can view statistics for your queries through the `pg_stat_statements`
+extension, which provides a `pg_stat_statements` view. The recorded statistics
+include the time spent planning and executing each query; the number of blocks
+hit, read, and written; and more.
+
+You can query the `pg_stat_statements` view as you would any PostgreSQL view.
+The full view includes superuser queries, which are used by Timescale Cloud to
+manage your service in the background. To limit the results to only your
+queries, filter by the current user.
+
+Connect to your database using a PostgreSQL client, such as [`psql`][psql], and
+run:
+```sql
+SELECT * FROM pg_stat_statements WHERE pg_get_userbyid(userid) = current_user;
+```
+
+<highlight type="important">
```

Comment:
May be this can go in the begining para of ## Query the pg_stat_statements view.